doc/index.rst

```diff
@@ -76,9 +72,9 @@ follows:
 
     dev = qml.device('qiskit.aer', wires=2, backend='unitary_simulator')
 
-PennyLane chooses the ``qasm_simulator`` as the default backend if no backend is specified.
-For more details on the ``qasm_simulator``, including available backend options, see
-`Qiskit Qasm Simulator documentation <https://qiskit.org/ecosystem/aer/stubs/qiskit_aer.QasmSimulator.html>`_.
+PennyLane chooses the ``aer_simulator`` as the default backend if no backend is specified.
+For more details on the ``aer_simulator``, including available backend options, see
+`Qiskit Aer Simulator documentation <https://qiskit.github.io/qiskit-aer/stubs/qiskit_aer.AerSimulator.html#qiskit_aer.AerSimulator.run>`_.
```

Comment:
Qasm simulator has been removed already. However, for the ibmq devices, the default behaviour cannot actually use the local aer simulators. Right now the default behaviour for the ibmq devices is already broken.

Reply:
Are we updating the behaviour here? Have we already updated it in a separate PR? Or does the deleted text accurately describe what the devices currently do?

Reply:
We need to update the behaviour for the ibmq devices since the ibmq devices use the ibmq_qasm_simulator as their default backend. For qiskit.aer and qiskit.remote the behaviour was already [updated in new_device_feature_branch](https://github.com/PennyLaneAI/pennylane-qiskit/pull/521)

Reply:
Oh wait, I just realized this is for qiskit.aer devices. Well, regardless, this was updated to use AerSimulator in that PR so this would be reflecting that change.

Reply:
Yeah just double checked. The default behaviour of qiskit.aer is to use AerSimulator as its backend. So... I suppose this is an accidental but correct change of the documentation to reflect reality.

Reply:
Ok. After some further investigation, what's going on here is that `qiskit.basicaer` used to default to `QasmSimulator` but `qiskit.aer` has always defaulted to `Aer_Simulator`. The Index.rst here was describing the behaviour of the `basicaer` device. With the removal of `basicaer`, we are now describing the behaviour of the `qiskit.aer`, which defaults to `Aer_Simulator`. World is round.

Reply:
We'll worry about the details when the docs update comes later anyway, I suppose, seems good for now